File: 2024/10 - October/26. Occurance of an Integer in a linked list/Solution.cpp
class Solution 
{
  public:
    int count(struct Node* head, int key) 
    {
        int res = 0;
        
        if(head == NULL) return res;
        
        while(head != NULL)
        {
            if(head->data == key)
            {
                res++;
            }
            head = head->next;
        }
        
        return res;
    }
};

File: 2024/10 - October/26. Occurance of an Integer in a linked list/Solution.java
class Solution 
{
    public static int count(Node head, int key) 
    {
        int res = 0;
        
        if(head == null) return res;
        
        while(head != null)
        {
            if(head.data == key)
            {
                res++;
            }
            head = head.next;
        }
        
        return res;
    }
}
